bin/build-locales

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+/* eslint-disable no-var, no-console */
+// We can't use strict mode because po2json causes
+// referenceErrors.
+
+
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const config = require('config');
+const po2json = require('po2json');
+const glob = require('glob');
+const shelljs = require('shelljs');
+const chalk = require('chalk');
+
+const appName = config.get('appName');
+
+if (!appName) {
+  console.log(
+    chalk.red('Please specify the appName with NODE_APP_INSTANCE'));
+  process.exit(1);
+}
+
+function createLockFile(lockFilePath) {
+  const fd = fs.openSync(lockFilePath, 'w');
+  fs.closeSync(fd);
+}
+
+function removeLockFile(lockFilePath) {
+  fs.unlinkSync(lockFilePath);
+}
+
+function lockFileExists(lockFilePath) {
+  var fileExists = false;
+  try {
+    fs.statSync(lockFilePath);
+    fileExists = true;
+  } catch (e) {
+    if (e.code !== 'ENOENT') {
+      throw e;
+    }
+  }
+  return fileExists;
+}
+
+const lockFileName = `${appName}-build-locales.lock`;
+const lockFilePath = path.join(os.tmpdir(), lockFileName);
+const localeDir = path.join(__dirname, '../locale');
+const poFiles = glob.sync(`${localeDir}/**/${appName}.po`);
+const dest = path.join(__dirname, '../src/locale/');
+
+if (lockFileExists(lockFilePath)) {
+  throw new Error(`Lock file exists at: ${lockFilePath} Aborting!`);
```

Comment:
Earlier you logged an error message and exited with `1` should this do something similar?